### ISSUE TITLE
nuget.xplat list errors when there is msbuild sdk

### ISSUE THREAD

**Issue by wli3** [NuGet/Home#7802]
[msbuild sdk](https://github.com/Microsoft/MSBuildSdks/tree/master/src/CentralPackageVersions)
```
λ dotnet list package
error: Sequence contains more than one element
```

### Sample Project
```
<Project Sdk="Microsoft.NET.Sdk">

  <Sdk Name="Microsoft.Build.CentralPackageVersions" Version="2.0.1" />

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>netcoreapp2.1</TargetFramework>
  </PropertyGroup>

</Project>

```

**Comment by danstur** [comment]
Is there any workaround for this problem?

**Comment by SimonCropp** [comment]
i am also getting it with `Microsoft.NET.Test.Sdk` version 16.1.1

**Comment by nkolev92** [comment]
It's possible that other work in this space fixed this issue. 
The bug would've been fixed during the current release. 

@heng-liu can you verify please.

**Comment by SimonCropp** [comment]
@nkolev92 what current release do you mean ? I am running the current sdk 2.2.300 and still get the issue

**Comment by nkolev92** [comment]
By current release, I'm referring to the one actively under development, so that's 2.2.400. 
Specifically the root cause could be the same as https://github.com/NuGet/Home/issues/8154.

**Comment by SimonCropp** [comment]
on a side note, it would be better if the `catch` that writes that error gave the full stack trace

**Comment by SimonCropp** [comment]
is there a preview release that fix is included in? so i can smoke test it

**Comment by nkolev92** [comment]
I'd imagine increasing the verbosity would give you the full stack trace. 
Can you try that?

**Comment by nkolev92** [comment]
Quickly looking at commit [tags](https://github.com/NuGet/NuGet.Client/commit/79fafc3d7ec0bd8e303ac07d133ca720a23dd4e3), it suggests that 2.2.400 P3 might contain it, so the preview that shipped last week.

**Comment by SimonCropp** [comment]
i dont think dotnet list supports verbosity

```
dotnet list package -h

Usage: dotnet list <PROJECT | SOLUTION> package [options]

Arguments:
  <PROJECT | SOLUTION>   The project or solution file to operate on. If a file is not specified, the command will search the current directory for one.

Options:
  -h, --help                                Show command line help.
  --outdated                                Lists packages that have newer versions.
  --framework <FRAMEWORK | FRAMEWORK\RID>   Chooses a framework to show its packages. Use the option multiple times for multiple frameworks.
  --include-transitive                      Lists transitive and top-level packages.
  --include-prerelease                      Consider packages with prerelease versions when searching for newer packages. Requires the '--outdated' option.
  --highest-patch                           Consider only the packages with a matching major and minor version numbers when searching for newer packages. Requires the '--outdated' option.
  --highest-minor                           Consider only the packages with a matching major version number when searching for newer packages. Requires the '--outdated' option.
  --config <CONFIG_FILE>                    The path to the NuGet config file to use. Requires the '--outdated' option.
  --source <SOURCE>                         The NuGet sources to use when searching for newer packages. Requires the '--outdated' option.
```

**Comment by nkolev92** [comment]
Yeah, that's true, that totally slipped my mind :( 
Neither of the dotnet add/list commands do that.

**Comment by SimonCropp** [comment]
so can we  get the full error (including stack trace) in the case of an unhandled exception. IMO that should not need changes in verbosity to see that info

**Comment by nkolev92** [comment]
All the exceptions go through an exception logger, which can limit the verbosity, so not sure that's possible, with the current builds.

**Comment by jtsom** [comment]
Seeing this error with a simple Azure Functions project:
```
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <TargetFramework>netcoreapp2.2</TargetFramework>
    <AzureFunctionsVersion>v2</AzureFunctionsVersion>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="Microsoft.Azure.DocumentDB.Core" Version="2.5.1" />
    <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="1.0.29" />
  </ItemGroup>
  <ItemGroup>
    <None Update="host.json">
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
    </None>
    <None Update="local.settings.json">
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
      <CopyToPublishDirectory>Never</CopyToPublishDirectory>
    </None>
  </ItemGroup>
</Project>
```

```
$ dotnet list package
error: Sequence contains more than one element
```

```
$ dotnet --version
2.2.300
```

**Comment by jirkadanek** [comment]
Works for me with `dotnet --version` of `2.2.401` (and runtime of `2.2.6`). Good to see the fix roiling out.

**Comment by nkolev92** [comment]
Just re verified. 
Fixed in 2.2.400, 

Changing the milestone as 5.2 and closing.
